examples/compiled/concat_bar_layer_circle.vg.json

```diff
@@ -32,11 +32,17 @@
           "type": "formula",
           "expr": "toNumber(datum[\"Rotten_Tomatoes_Rating\"])",
           "as": "Rotten_Tomatoes_Rating"
-        },
+        }
+      ]
+    },
+    {
+      "name": "data_2",
+      "source": "data_1",
+      "transform": [
         {
           "type": "extent",
           "field": "IMDB_Rating",
-          "signal": "concat_0_layer_0_bin_maxbins_10_IMDB_Rating_extent"
+          "signal": "concat_0_layer_1_bin_maxbins_10_IMDB_Rating_extent"
```

Comment:
@domoritz this is somehow still re-ordered.

Reply:
This may be due to swapping nodes. We do not guarantee order preservation there.